src/index.d.ts

```diff
@@ -1,11 +1,38 @@
 import * as aws from "aws-sdk";
 import { TestInterface } from "ava";
 
+export interface Config {
```

Comment:
If we really do end up exposing this then it seems to me that we should expose the real upstream types (applies to types below too).

Reply:
This is exposing the real types.

Reply:
It's exposing some of them, but many of the types here represent trimmed/re-implemented versions of the upstream types. For example, wouldn't `ServiceConfigurationOptions` be what is accepted by the client constructors?

Reply:
I removed this interface, and updated the ones that are exposed in the context for DynamoDB and Kinesis to use the interface you suggested.